menu/menu_displaylist.c

```diff
@@ -104,14 +104,6 @@ static enum msg_hash_enums new_type     = MSG_UNKNOWN;
  * in playlists */
 #define PL_LABEL_SPACER_DEFAULT "   |   "
 #define PL_LABEL_SPACER_RGUI    " | "
-#if defined(__APPLE__)
-/* UTF-8 support is currently broken on Apple devices... */
-#define PL_LABEL_SPACER_OZONE   "   |   "
-#else
-/* <EM SPACE><BULLET><EM SPACE>
- * UCN equivalent: "\u2003\u2022\u2003" */
-#define PL_LABEL_SPACER_OZONE   "\xE2\x80\x83\xE2\x80\xA2\xE2\x80\x83"
-#endif
 #define PL_LABEL_SPACER_MAXLEN 37
```

Comment:
Could you reduce the size of `PL_LABEL_SPACER_MAXLEN` accordingly?

Reply:
What would be the new size? 7 ?

Reply:
Almost - you forgot the null terminator! Should be 8, I think.

(I mean, you could just leave it as-is, but might as well save a few bytes of RAM)

Reply:
I was not sure of the meaning of that value because 37 seems oddly large for the ozone spacer. Fixing it now

Reply:
Actually, you're right - I just realised that I set the size too large to begin with... :disappointed: 

But you've fixed it now, so it's all good!